test/evergreen.yml

```diff
@@ -307,7 +307,7 @@ functions:
               -e 's|gcp_client_x509_cert_url|${gcp_sdk_ext_client_x509_cert_url}|'  ../test/evergreen/gcp_auth.json > $file
           export GOOGLE_APPLICATION_CREDENTIALS="$file"
 
-          virtualenv -p python3 venv
+          ${configure_env_vars|} virtualenv -p python3 venv
```

Comment:
I'd prefer for this to be `test_env_vars` but the change is causing some issues since the `ubuntu2004` variant doesn't set `PATH=/opt/mongotoolchain/v4/bin:$PATH` and and updating it to do so is impacting other tests.
For now I'd prefer to stick with `configure_env_vars` and handle it in the proposed new ticket

Reply:
(Non blocking) Is it would just be explicit to PATH in the short term and then addressing it more in the future ticket?

Reply:
SGTM. Updated in [988abe2](https://github.com/wiredtiger/wiredtiger/pull/9346/commits/988abe29d374732f21c473652e77179bee17abee)